lib/HTTP/Server/Tiny.pm6

```diff
@@ -261,6 +261,7 @@ my class HTTP::Server::Tiny::Handler {
             if $result ~~ Promise {
                 return $result.result;
             }
+            warn "P6W app must return Promise";
```

Comment:
P6W specs still support old psgi protocol, so returning Promise is not "`must`", in P6W specification.
https://github.com/zostay/P6W#4-application-protocol-implementation

(Btw how Crust will handle those response protocols? I think it's better not drop old psgi protocols right now)